animeal/src/Flows/Main/Modules/Home/FeedingPointDetails/Model/FeedingPointDetailsModel.swift

```diff
@@ -90,7 +90,11 @@ final class FeedingPointDetailsModel: FeedingPointDetailsModelProtocol, FeedingP
         let history = try await context.feedingPointsService.fetchFeedingHistory(for: fullFeedingPoint.identifier)
         guard !history.isEmpty else { return [] }
 
-        let sortedByDateHistory = history.sorted(by: { $0.updatedAt > $1.updatedAt })
+//        let sortedByDateHistory = history.sorted(by: { $0.updatedAt > $1.updatedAt })
+        let sortedByDateHistory = history.sorted { item1, item2 in
```

Comment:
please check for higher order short hand syntax here and try.
https://www.appcoda.com/higher-order-functions-swift/#Sorted